S2VX.Game/Editor/CommandPanel/CommandPanelValueInput.cs

```diff
@@ -61,7 +87,9 @@ public class CommandPanelValueInput : Container {
 
         // Bindings need to be set up early so that they can trigger before
         // Load() happens
-        public CommandPanelValueInput() {
+        public CommandPanelValueInput(Func<double> currentTimeDelegate = null) {
+            CurrentTimeDelegate = currentTimeDelegate;
+            BtnApplyCurrentTime.Action = ApplyCurrentTime;
```

Comment:
If you want, I don't think you need to store `currentTimeDelegate` as a property. You can update `ApplyCurrentTime` to take the delegate.

```suggestion
            BtnApplyCurrentTime.Action = () => ApplyCurrentTime(currentTimeDelegate);
```

Reply:
Implemented